nway/nway_matching.py

```diff
@@ -411,22 +413,27 @@ def run(self):
 
         # generate N-way matching table
         matching_frame = self.gen_nway_table_with_redundancy()
-
         matching_table_nway_tmp = self.remove_nway_table_redundancy(
                 matching_frame)
-
         matching_table_nway_tmp = self.prune_matching_graph(
                 matching_table_nway_tmp)
-
         self.matching_table_nway = self.add_remaining_cells(
                 matching_table_nway_tmp)
-
         self.logger.info("Nway matching is done!")
 
         output_dict = self.create_output_dict()
         self.output(output_dict, indent=2)
         self.logger.info("wrote {}".format(self.args['output_json']))
 
+        # Generate nway match summary plots
```

Comment:
can this get bumped up above the nway_matching output_json writing, such that the written plot(s) have their paths specified in one output json for the entire job.
also, given Marina's comments about pdf vs. png, I think the updated output json for nway_matching should have either these 2 keys: `fraction_matched_plot: <Path(.png)>` and `overlay_plot: <Path(.png)>`
OR
it could point to the output json of `NwaySummary` which could have an output schema validating the output paths of those 2 plots.
My rationale is: when undertaking the survey task of hundreds of containers, I only need to find on isilon the output json of the nway job, and, pointers to plots are found therein.

Reply:
This is done, added 3 new fields to the output.json of nway that point to the paths of the summary *.png plots.